GitUI/Editor/RebaseTodoHighlightingStrategy.cs

```diff
@@ -17,15 +17,15 @@ internal sealed class RebaseTodoHighlightingStrategy : GitHighlightingStrategyBa
         d, drop = remove commit
         */
 
-        private static readonly Dictionary<char, (string longForm, HighlightColor color)> _commandByFirstChar = new()
+        private static readonly Dictionary<char, (string longForm, HighlightColor color, string[] options)> _commandByFirstChar = new()
         {
-            { 'p', ("pick", new HighlightColor(Color.Black, bold: true, italic: false)) },
-            { 'r', ("reword", new HighlightColor(Color.Purple, bold: true, italic: false)) },
-            { 'e', ("edit", new HighlightColor(Color.Black, bold: true, italic: false)) },
-            { 's', ("squash", new HighlightColor(Color.DarkBlue, bold: true, italic: false)) },
-            { 'f', ("fixup", new HighlightColor(Color.LightCoral, bold: true, italic: false)) },
-            { 'x', ("exec", new HighlightColor(Color.Black, bold: true, italic: false)) },
-            { 'd', ("drop", new HighlightColor(Color.Red, bold: true, italic: false)) }
+            { 'p', ("pick", new HighlightColor(Color.Black, bold: true, italic: false), Array.Empty<string>()) },
+            { 'r', ("reword", new HighlightColor(Color.Purple, bold: true, italic: false), Array.Empty<string>()) },
+            { 'e', ("edit", new HighlightColor(Color.Black, bold: true, italic: false), Array.Empty<string>()) },
+            { 's', ("squash", new HighlightColor(Color.DarkBlue, bold: true, italic: false), Array.Empty<string>()) },
+            { 'f', ("fixup", new HighlightColor(Color.LightCoral, bold: true, italic: false), new[] { "-C", "-c" }) },
```

Comment:
https://git-scm.com/docs/git-commit#Documentation/git-commit.txt--Cltcommitgt
```
-C <commit>
--reuse-message=<commit>

    Take an existing commit object, and reuse the log message and the authorship information (including the timestamp) when creating the commit.
```
IIUIC, this will change the author of the commit, too. I would find this surprising.

Reply:
I was confused by this, too, because in my tests the authorship of the original commit was kept. But then I've re-read the documentation for the `--fixup` option (just below the `-C` option you linked to) and it says:
_Neither "fixup!" nor "amend!" commits change authorship of `<commit>` when applied by `git rebase --autosquash`._